frontend/src/metabase/visualizations/components/LegendItem.jsx

```diff
@@ -32,6 +35,10 @@ export default class LegendItem extends Component {
     showDotTooltip: true,
   };
 
+  getRootElement() {
+    return this.rootRef.current;
+  }
```

Comment:
Expose the `span`'s DOM element for use in `onHoverChange` in LegendVertical and LegendHorizontal, to trigger logic when hovering on chart legends.